package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "npm-install-webpack-plugin",
-  "version": "4.0.4",
+  "version": "4.0.5",
```

Comment:
Please revert this, it is the maintainer/releaser responsibility to do and we have a standard process for this in place already :)